MUONTPAR/iocBoot/iocMUONTPAR-IOC-01/st.cmd

```diff
@@ -19,6 +19,7 @@ MUONTPAR_IOC_01_registerRecordDeviceDriver pdbbase
 ## Load our record instances
 dbLoadRecords("$(TOP)/db/muon_tpar.db","P=$(MYPVPREFIX)$(IOCNAME):,TPAR_FILE=$(TPAR_FILE=),TPAR_FILE_PV_NAME=TPAR_FILE")
 dbLoadRecords("$(TOP)/db/muon_tpar.db","P=$(MYPVPREFIX)$(IOCNAME):,TPAR_FILE=$(BOOSTER_TPAR_FILE=),TPAR_FILE_PV_NAME=BOOSTER_TPAR_FILE")
+dbLoadRecords("$(TOP)/db/muon_tpar.db","P=$(MYPVPREFIX)$(IOCNAME):,TPAR_FILE=$(BOOSTER_TYPE=),TPAR_FILE_PV_NAME=BOOSTER_TYPE")
 
```

Comment:
Should this have OXF13 as a default macro value?

Reply:
Added